src/route-config.ts

```diff
@@ -32,7 +33,7 @@ export type RouteDef = {
 
 type RoutesFn = (t: TFunction) => RouteDef[];
 
-export const routes: RoutesFn = (t: TFunction) => [
+export const routes: RoutesFn = (t: any) => [
```

Comment:
don't think this change is needed right?

Reply:
@edewit this was needed because i wanted to be able to pass null, otherwise it would fail

Reply:
why not be more precise and make it:  
```suggestion
export const routes: RoutesFn = (t: TFunction|null) => [
```